README.md

```diff
@@ -259,9 +259,170 @@ Otherwise the client will return an error `err`:
 </table>
 </details>
 
+### Letter
+
+```javascript
+notifyClient
```

Comment:
Inconsistent indentation here. Mixing tabs and spaces?

Reply:
ok, will fix